tests/plugins/test_orchestrate_build.py

```diff
@@ -183,9 +188,11 @@ def test_orchestrate_build(tmpdir):
                 'platforms': ['x86_64'],
                 'build_kwargs': make_worker_build_kwargs(),
                 'osbs_client_config': str(tmpdir),
+                'worker_build_image': worker_build_image
```

Comment:
Maybe only set `worker_build_image` in `args` if it's not `None`? That way you can verify that it default to `None` when not provided.